### PR TITLE
Feat/cleanup and 3IPS

### PR DESCRIPTION
**Pull request by oed**

### CODE REVIEW

File: 3IPs/3ip-0.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+3ip: 0
+title: 3IP Purpose and Guidelines
+status: Active
+author: Joel Thorstensson <oed@3box.io>
+created: 2019-01-16
+---
+
+## What is a 3IP?
+
+3IP stands for 3Box Improvement Proposal. A 3IP document contains information describing new features or other types of changes to the 3Box community. They are intended to be similar to EIPs and BIPs of Ethereum and Bitcoin respectively, but we are starting of with less formalities for now.
```
Comment (michaelsena) [inline 3IPs/3ip-0.md]: typo // change to: "..., but 3IPs won't maintain the same level of formality as the others – for now."

File: 3IPs/3ip-0.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+## Creating a 3IP
+Simply copy [3ip-X.md](./3ip-X.md) and start writing. Once you have a proposal ready make a PR to this repo adding your file. The number of your PR becomes the number of the 3ip.
```
Comment (michaelsena) [inline 3IPs/3ip-0.md]: Change to: "... Once your proposal is ready, add your file to this repo by making a PR. The number of your PR will become the number of your 3IP."

**Comment by oed** [comment]
@michaelsena Updated with your edits.

**Comment by michaelsena** [comment]
Thanks @oed I got sidetracked yesterday so couldn't finish the review. Will get to it today hopefully!

File: README.md [hunk 3]
```diff
@@ -69,15 +69,9 @@ The [3box.io](https://3box.io) dapp is how users manage their 3Box data, includi
 
 <br>
 
-# More Information
-
 ## Project Goals
```
Comment (michaelsena) [inline README.md]: Let's make "Project Goals" a H1, instead of a H2

File: devops/service-endpoints.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# 3Box service endpoints
```
Comment (michaelsena) [inline devops/service-endpoints.md]: Update capitalization to "3Box Service Endpoints"

File: devops/service-endpoints.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+3Box provides a set of services that makes the 3Box network more resilient. Here all of the service endpoints are documented. To see APIs and documentation please refer to the repo for the given service.
```
Comment (michaelsena) [inline devops/service-endpoints.md]: Update first sentence to: "3Box provides a set of services that make the 3Box network more resilient, convenient, scalable, and performant."

**Pull request merged**